lib/v2/shmtu/portal.js

```diff
@@ -22,6 +22,46 @@ const processFeed = (list, caches) =>
             caches.tryGet(item.link, async () => {
                 const detail = await loadDetail(item.link);
                 item.description = detail.body.und[0].safe_value;
+                let images = detail.field_image;
+                if (images.length !== 0 && Object.keys(images).length !== 0) {
+                    images = images.und;
+                    item.description += `
+<span style="font-size:19px">
```

Comment:
Please use `art-template` for custom HTML contents
docs: https://docs.rsshub.app/joinus/script-standard.html#template